pkgs/development/python-modules/humann/default.nix

```diff
@@ -0,0 +1,66 @@
+{
+  lib,
+  python,
+  buildPythonPackage,
+  callPackage,
+  fetchzip,
+  fetchFromGitHub,
+  setuptools,
+  biom-format,
+  bowtie2,
+  diamond,
+  samtools,
+  humann-db,
```

Comment:
```suggestion
  humann-db ? null,
```

Reply:
Is there any scenario in which `humann-db` will not be passed?

Reply:
Checked with the Discord server, this should be unneccesary.